apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue

```diff
@@ -105,7 +105,8 @@ const getPluginMetadata = async () => {
         if (!props.hasMetadata) {
             const res = await fetchGetPluginMetadata();
             state.schema = res.metadata?.options_schema ?? {};
-            if (!state.schema) {
+            const isSchemaEmpty = !Object.keys(state.schema ?? {}).length;
```

Comment:
nitpick: What about declare it as computed state since it is duplicated with the 86 line. :)